src/screens/TopicPageScreen/elements/NavHeader.js

```diff
@@ -41,14 +49,13 @@ const NavHeader = (props) => {
     onCancelToken,
     placeholderText,
     setIsFirstTimeOpen,
-    getSearchLayout
+    getSearchLayout,
+    opacityImage
   } = props;
   const navigation = useNavigation();
   const coverPath = topicDetail?.cover_path || null;
-  const [domainHeight, setDomainHeight] = React.useState(0);
-  const [searchHeight, setSearchHeight] = React.useState(0);
-  const {top} = useSafeAreaInsets();
-  const topPosition = Platform.OS === 'ios' ? top : 0;
+  const insets = useSafeAreaInsets();
+  const {width: displayWidth} = useWindowDimensions();
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The removal of the `onDomainLayout` and `onSearchLayout` callbacks, which were used to calculate the height of certain elements, could potentially cause layout issues if these heights are still being used elsewhere in the code. Please ensure that these changes do not affect the layout or functionality of the component.



<!-- This is an auto-generated comment by OSS CodeRabbit -->